bundles/org.openhab.ui.dashboard/ESH-INF/i18n/dashboard_fi.properties

```diff
@@ -0,0 +1,27 @@
+#X-Generator: crowdin.com
```

Comment:
See also here imho this should not contains english texts

Reply:
I was in the process of setting up crowdin with the right configuration, should be okay now.

Reply:
Great!